src/index.js

```diff
@@ -635,6 +635,9 @@ function _renderToString(
 					}
 				} else if (HTML_LOWER_CASE.test(name)) {
 					name = name.toLowerCase();
+					if (name === 'spellcheck') {
```

Comment:
Not that it necessarily needs to be done here, but I think there's at least a handful of attrs/props that could need (depending on use circumstances) to be preserved as falsy.

Might be something we need to add another regex constant for.

Reply:
Yeah, good point. Let's keep out an eye for that.